sdk/storage/storage-blob/samples/storage-blob-resize-function/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "function-image-upload-resize-node",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@azure/storage-blob": "^12.1.2",
```

Comment:
Maybe better to reference to a latest released version.

Reply:
Storage Blob SDK is a latest released version now.